File: Leet Code/30 Days of Code/week_01/counting_numbers.py
class Solution:
    def countElements(self, arr: List[int]) -> int:
        numbers = [0]*1002
        for i in arr:
            numbers[i] += 1 
        count = 0
        for i in arr:
            if numbers[i+1] > 0:
                count += 1 
            
        return count

File: Leet Code/30 Days of Code/week_01/counting_numbers.cpp
#include<iostream>
#include<set>
#include<vector>
using namespace std;
class Solution{
    public:
        int countElements(vector<int>& arr){
            set<int> s;
            for(int x:arr){
                s.insert(x)
            }
            int count = 0;
            for(int x:arr){
                if(s.count(x+1)==1){
                    count++;
                }
            }
            return count;
        }
}
